DEV.md

```diff
@@ -160,6 +160,28 @@ When developing migrations the best way to test them on a clean database is:
 make test-migrate
 ```
 
+#### Create a new migration
+
+Create the migration file
+
+```bash
+node_modules/.bin/migrate-mongo create <migration-name> \
+  -f src/migrations/migrate-mongo-config.js
+```
+
+Write the migration in the newly created migration file and then test/run with the following:
+```bash
+# Migrate
+node_modules/.bin/migrate-mongo up \
```

Comment:
instead of this use `npx migrate-mongo up ...` (applies for all cmds)

Reply:
Don't think we have `npx` as a mandatory dev dependency... should probably move to nix based dev envs at some point.

Reply:
`Since npm version 5.2.0 npx is pre-bundled with npm`

Reply:
Changed it to `npx`... `yarn` had some issue with node versions when I tried to run with it https://github.com/GaloyMoney/galoy/compare/1da5b8504fd7c5f01955a13eb91cec00841cbf01..0d7142e1bf1e3d38fd75c5463c7041faeb5af44b